src/locale/defaults.h

```diff
@@ -279,6 +279,9 @@
 #ifndef D_STR_ID
 #define D_STR_ID "Id"
 #endif  // D_STR_ID
+#ifndef D_STR_DRY_GRADE
+#define D_STR_DRY_GRADE "Dry grade"
```

Comment:
```c++
#define D_STR_DRY_GRADE D_STR_DRY " grade"
```
and put it in the compound word section. That way some of the other language files have less work to do to change it for another language. etc.

Also, do we really need to have "Dry grade: 2" etc. Could we live with "Dry: 2" and save adding another string etc.? (your call, just a suggestion to save space)